src/chains/migrations/0019_add_safe_apps_rpc.py

```diff
@@ -0,0 +1,44 @@
+# Generated by Django 3.2.5 on 2021-07-23 15:23
+
+from django.db import migrations, models
+from django.db.models import F
+
+
+def copy_rpc_fields(apps, schema_editor):
+    chain_model = apps.get_model("chains", "chain")
+    db_alias = schema_editor.connection.alias
+    chain_model.objects.using(db_alias).all().update(
+        safe_apps_rpc_uri=F("rpc_uri"),
+        safe_apps_rpc_authentication=F("rpc_authentication"),
+    )
```

Comment:
⚠️ Currently this is a not reversible migration. Revert implementation should be simple. Check: https://docs.djangoproject.com/en/3.2/ref/migration-operations/#runpython